sdk/analytics/src/main/java/com/klaviyo/analytics/networking/ApiClient.kt

```diff
@@ -2,8 +2,14 @@ package com.klaviyo.analytics.networking
 
 import com.klaviyo.analytics.model.Event
 import com.klaviyo.analytics.model.Profile
+import com.klaviyo.analytics.networking.requests.ApiRequest
 
-internal interface ApiClient {
+typealias ApiObserver = (request: ApiRequest) -> Unit
+
+/**
+ * Defines public API of the network coordinator service
+ */
+interface ApiClient {
```

Comment:
While the interface is public, the implementation will still be inaccessible like all other services except via `core`